Sources/NIO/EventLoop.swift

```diff
@@ -608,6 +608,35 @@ extension EventLoop {
         }
     }
 
+    /// Creates and returns a new `EventLoopFuture` that is marked as succeeded or failed with the value held by `result`.
+    ///
+    /// - Parameters:
+    ///   - result: The value that is used by the `EventLoopFuture`
+    /// - Returns: A completed `EventLoopFuture`.
+    @inlinable
+    public func makeCompletedFuture<Success>(_ result: Result<Success, Error>, file: StaticString = #file, line: UInt = #line) -> EventLoopFuture<Success> {
```

Comment:
these don't need the `file`/`line`, don't know why `makeFailed/Succeeded` want them. The `file`/`line` is only interesting for promises so we can tell you where you created an unfulfilled one.